src/enclave/main.cpp

```diff
@@ -4,6 +4,10 @@
 #include "../ds/spinlock.h"
 #include "enclave.h"
 
+#ifdef PBFT
+#  include "../src/consensus/pbft/pbftglobals.h"
```

Comment:
Should this be `#include "consensus/pbft/pbftglobals.h"`?

Reply:
Actually fixing this get undone by clang-format